src/main/scala/battleship/routes/gameActor.scala

```diff
@@ -0,0 +1,84 @@
+package battleship.routes
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import battleship.PlayerActor.{GetNextShot, PlaceBoats}
+import battleship.game._
+import battleship.routes.gameActor.{StartGame, ThisIsBoatSetup, ThisIsNextMove}
+
+/**
+  * Created by m06f947 on 3-8-2017.
+  */
+
+
+
+object gameActor {

```

Comment:
Conventie is dat een object en class met een hoofdletter begint. Tip: gebruik de refactor feature van Intellij: Shift+F6 als je met de cursor op `gameActor` staat